packages/@aws-cdk/aws-efs/lib/access-point.ts

```diff
@@ -0,0 +1,168 @@
+   * the file's mode bits.
+   */
+  readonly permissions: string;
+}
+
+/**
+ * Represents the PosixUser
+ */
+export interface PosixUser {
+  /**
+   * The POSIX user ID used for all file system operations using this access point.
+   */
+  readonly uid: string;
+
+  /**
+   * The POSIX group ID used for all file system operations using this access point.
+   */
+  readonly gid: string;
+
+  /**
+   * Secondary POSIX group IDs used for all file system operations using this access point.
+   *
+   * @default - None
+   */
+  readonly secondaryGids?: string[];
+}
+
+/**
+ * Properties for the AccessPoint
+ */
+export interface AccessPointProps {
+  /**
+   * The efs filesystem
+   */
+  readonly fileSystem: IFileSystem;
+
+  /**
+   * Specifies the POSIX IDs and permissions to apply when creating the access point's root directory. If the
+   * root directory specified by `path` does not exist, EFS creates the root directory and applies the
+   * permissions specified here. If the specified `path` does not exist, you must specify `createAcl`.
+   *
+   * @default - None
+   */
+  readonly createAcl?: CreationInfo;
+
+  /**
+   * Specifies the path on the EFS file system to expose as the root directory to NFS clients using the access point
+   * to access the EFS file system
+   *
+   * @default '/'
+   */
+  readonly path?: string;
+
+  /**
+   * The full POSIX identity, including the user ID, group ID, and any secondary group IDs, on the access point
+   * that is used for all file system operations performed by NFS clients using the access point.
+   *
+   * Specify this to enforce a user identity using an access point.
+   *
+   * @see - [Enforcing a User Identity Using an Access Point](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html)
+   *
+   * @default - user identity not enforced
+   */
+  readonly posixUser?: PosixUser;
+}
+
+/**
+ * Represents the AccessPoint
+ */
+export class AccessPoint extends Resource implements IAccessPoint {
+  /**
+   * Import an existing Access Point
+   */
+  public static fromAccessPointId(scope: Construct, id: string, accessPointId: string): IAccessPoint {
+    class Import extends Resource implements IAccessPoint {
+      public readonly accessPointId = accessPointId;
+      public readonly accessPointArn = Stack.of(scope).formatArn({
+        service: 'elasticfilesystem',
+        resource: 'access-point',
+        resourceName: accessPointId,
+      });
+    }
+    return new Import(scope, id);
+  }
+
+  /**
+   * The ARN of the Access Point
+   * @attribute
+   */
+  public readonly accessPointArn: string;
+
+  /**
+   * The ID of the Access Point
+   * @attribute
+   */
+  public readonly accessPointId: string;
+
+  constructor(scope: Construct, id: string, props: AccessPointProps) {
+    super(scope, id);
+
+    if (props.path && !props.createAcl) {
+      throw new Error('createAcl is required if path is specified');
+    }
```

Comment:
Why enforce this? From what I understood, if the path specified already exists on the EFS file system, `createAcl` is not needed. Did I misunderstand?

Reply:
Well, there is a case if user specifies the `path` but actually the path doesn't exist and `createAcl` not specified either, cloudformation will throws error. However, we can't check if the specified `path` really exists. Looks like we don't have to prevent this condition and simply tell user the specified `path` has to exist, right?

<img width="508" alt="image" src="https://user-images.githubusercontent.com/278432/85394997-0f94b000-b582-11ea-8f2d-970ffb4af84c.png">

Reply:
Just removed the check. Let's see if it looks better now.

Reply:
You're right. Unfortunately, it's not something we can validate in the CDK.